packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -240,7 +241,9 @@ const Services = WebexPlugin.extend({
    * @param {ValidateUserPTO} - The parameter transfer object.
    * @returns {ValidateUserRTO} - The return transfer object.
    */
-  validateUser({email, reqId = 'WEBCLIENT', forceRefresh = false}) {
+  validateUser({
+    email, reqId = 'WEBCLIENT', forceRefresh = false, activationOptions = {}
```

Comment:
new line params for readability
```suggestion
    email,
    reqId = 'WEBCLIENT',
    forceRefresh = false,
    activationOptions = {}
```